package-lock.json

```diff
@@ -20,10 +20,10 @@
       "dev": true,
       "requires": {
         "@babel/types": "7.0.0-beta.44",
-        "jsesc": "^2.5.1",
-        "lodash": "^4.2.0",
-        "source-map": "^0.5.0",
-        "trim-right": "^1.0.1"
+        "jsesc": "2.5.1",
```

Comment:
What version of npm are you using? We shouldn't be changing this file too much, and I think there might be just differences between different versions of npm

Reply:
This is what I get when I run `npm version`
{ 'mattermost-webapp': '0.0.1',
  npm: '5.6.0',
  ares: '1.13.0',
  cldr: '32.0.1',
  http_parser: '2.7.0',
  icu: '60.2',
  modules: '59',
  nghttp2: '1.29.0',
  node: '9.4.0',
  openssl: '1.0.2n',
  tz: '2017c',
  unicode: '10.0',
  uv: '1.18.0',
  v8: '6.2.414.46-node.17',
  zlib: '1.2.11' }

Reply:
Since the package-lock.json changes weren't needed for this PR, you're fine for now, but it may make your life easier if you update NPM so that package-lock.json isn't modified every time `npm install` is ran